src/TestSuite/Mutant/Runner.php

```diff
@@ -50,6 +49,9 @@ class Runner
      */
     private $observers = [];
 
+    /**
+     * @param string $baseDirectory
```

Comment:
How so ? $baseDirectory **is** a string.... or are you referring to something else ?